codec-http2/src/main/java/io/netty/handler/codec/http2/AbstractHttp2StreamChannel.java

```diff
@@ -91,7 +94,7 @@ public boolean isOpen() {
 
     @Override
     public boolean isActive() {
-        return !closed;
+        return isOpen();
```

Comment:
No, (as documented in `Http2StreamChannelBootstrap` a child channel becomes active immediately after registration to an eventloop. At that point, it has not been assigned a stream identifier.

The reason for that is to stay compatible with HTTP Handlers, as else one would need to send a HEADERS frame before becoming active.

Thoughts?